core/src/send_transaction_service.rs

```diff
@@ -208,49 +217,58 @@ impl SendTransactionService {
                 info!("Transaction is rooted: {}", signature);
                 result.rooted += 1;
                 inc_new_counter_info!("send_transaction_service-rooted", 1);
-                false
+                return false;
+            }
+            if let Some((nonce_pubkey, durable_nonce)) = transaction_info.durable_nonce_info {
+                let nonce_account = working_bank.get_account(&nonce_pubkey).unwrap_or_default();
+                if !nonce_account::verify_nonce_account(&nonce_account, &durable_nonce) {
+                    info!("Dropping expired durable-nonce transaction: {}", signature);
+                    result.expired += 1;
+                    inc_new_counter_info!("send_transaction_service-expired", 1);
```

Comment:
Do we care about tracking this case with a separate metric?

Reply:
Off hand, I can't think of anything actionable based off such a metric.  Total nonced TXs in the queue might be interesting though.

Reply:
Added nonced-tx metric here: https://github.com/solana-labs/solana/pull/14308/commits/5994c0104785d8bb4a48acbc49693ca2841eea4f